src/adapter/services/LegacyPersistenceAdapter.js

```diff
@@ -39,8 +39,15 @@ export default class LegacyPersistenceAdapter {
         return this.openmct.objects.save(legacyDomainObject.useCapability('adapter'));
     }
 
-    readObject(keystring) {
-        let identifier = objectUtils.parseKeyString(keystring);
+    readObject() {
```

Comment:
Looks like this method [will always receive `space` and `key`](https://github.com/nasa/openmct/blob/master/docs/src/guide/index.md#persistence-service), and never a keystring. This was a misread on my part. So, we should just build an identifier object using those arguments explicitly.

ie.
``` JavaScript
readObject(space, key) {
    const identifier = {
        namespace: space,
        key: key
    }
    ///etc...
}
```

Reply:
Got it! updated!